server/errors_gen.go

```diff
@@ -74,16 +74,20 @@ func goFmt(file string) error {
 
 func checkDupes(errs []Errors) error {
 	codes := []int{}
+	highest := 0
 	for _, err := range errs {
 		codes = append(codes, err.ErrCode)
+		if highest < err.ErrCode {
+			highest = err.ErrCode
+		}
 	}
 
 	codeKeys := make(map[int]bool)
 	constKeys := make(map[string]bool)
 
 	for _, entry := range errs {
 		if _, found := codeKeys[entry.ErrCode]; found {
-			return fmt.Errorf("duplicate error code %+v", entry)
+			return fmt.Errorf("duplicate error code %+v, highest code is %d", entry, highest)
```

Comment:
Good, but I thought that you would have an option to the tool to tell us what are the highest codes for each "category". Can be added later, so not holding the merge for that.

Reply:
Indeed, once we have more than one cat I will add :P